src/main/java/com/datasonnet/badgerfish/XMLEventPipe.java

```diff
@@ -1,4 +1,4 @@
-package com.datasonnet;
+package com.datasonnet.badgerfish;
```

Comment:
This class has nothing to do with badgerfish

Reply:
I just repackaged everything related to the XML support into one package. We can rename it to "xml" or something like this. I'm open to suggestions.